### PR TITLE
Device code not device

### PR DESCRIPTION
**Pull request by anelachan**
Goal was to clean up the `device` column for device timespent. DP already has cleanup through its `device_code` column, however so using that instead.
- However will change uninformative device_code `'taco'` to `'appletv'`. (I found taco = appletv, appletv = apple tv classic.)
- Because `device_code` is already lowercase with no spaces, the `rename` function isn't needed.

### CODE REVIEW

File: features/session/query.py [hunk 1]
```diff
@@ -6,7 +6,7 @@ def get_session_sql(query_dt='', days_back=28):
     select
         user_id,
         session_id,
-        device,
+        case when device_code = 'taco' then 'appletv' else device_code end as device_code,
```
Comment (jlee5000) [inline features/session/query.py]: non-blocker, what's `taco`?

Comment (anelachan) [reply features/session/query.py]: it's apple TV (not classic). was obscured for some privacy reason apparently.

File: features/session/query.py [hunk 2]
```diff
@@ -27,25 +27,38 @@ def get_session_sql(query_dt='', days_back=28):
...
+        user_id,
+        session_id,
+        cast(session_start as date) session_start_date,
+        cast(unix_timestamp(session_end) - unix_timestamp(session_start) as float) / 60.0 as session_length_min
+    from
+        sessions
+
+) session_lengths
+where
+    -- remove sessions 8 hours or longer
```
Comment (jlee5000) [inline features/session/query.py]: would the session_length_min check time be useful as a parameter?  In the case if we need to check for 9 hours, or 6/7 hours. If not then 👍

Comment (anelachan) [reply features/session/query.py]: I don't think it's indicative of churn. We (will) have separate features for binge watching which might be significant but this most likely means your app is running accidentally.

**Pull request merged**